src/vue2-slider.vue

```diff
@@ -110,10 +110,10 @@
         </div>
       </template>
       <ul class="vue-slider-piecewise">
-        <li 
-          v-for="(piecewiseObj, index) in piecewiseDotWrap" 
-          class="vue-slider-piecewise-item" 
-          :style="[piecewiseDotStyle, piecewiseObj.style]" 
+        <li
```

Comment:
What value do these changes add to this PR?

Reply:
None, that's just my editor removing trailing whitespace.